brainframe_qt/ui/brainframe_app.py

```diff
@@ -90,10 +96,9 @@ def exec(self):
             self.splash_screen.showMessage(message)
 
             ExtensionLoader().load_extensions()
+            self.main_window.show()
```

Comment:
Just curious, does the order matter here?

```
self.main_window.show()
self.splash_screen.finish(self.main_window)
```

It used to be flipped

Reply:
It doesn't really matter, but [the docs](https://doc.qt.io/qt-5/qsplashscreen.html#details) have you `.finish` the splashscreen after `.show`ing the window